sycl/source/detail/pi.cpp

```diff
@@ -35,6 +28,25 @@ bool piUseBackend(PiBackend Backend) {
   return Backend == Use;
 }
 
+// Definitions of the PI dispatch entries, they will be initialized
+// at their first use with piInitialize.
+#define _PI_API(api) decltype(::api) * api = nullptr;
+#include <CL/sycl/detail/pi.def>
+
+// TODO: implement real plugins (ICD-like?)
+// For now this has the effect of redirecting to built-in PI OpenCL plugin.
+bool piInitialize() {
+  if (!piUseBackend(SYCL_BE_PI_OPENCL)) {
+    piDie("Unknown SYCL_BE");
+  }
+  #define _PI_API(api) \
+    extern const decltype(::api) * api##OclPtr; \
```

Comment:
Probably nicer to align the `\` vertically for macro definitions like it is done is the code around

Reply:
Maybe